### PR TITLE
Readme.md file updated

### PR DESCRIPTION
**Pull request by tanveerisonline**
### **Hi Code Reviewer 👋 👋**
**In this pull request, the following changes have been made:**

-  Splash screen - with login and sign-up buttons
-  When the user logs in, the categories page is presented
-  There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
-  User can see the name, icon, and the total amount of all the transactions that belong to that category.
-  When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
-  There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
-  For a given category, the list of transactions is presented, ordered by the most recent.
-  At the top of the page the user can see the total amount for the category
-  When the user clicks on the "Back" button (<), the user navigates to the home page.
-  Add sidebar navigation
-  Add unit and integration tests for the RoR application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,161 @@
...
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
+
+## video demo 🎥
+
+[video demo here](https://www.loom.com)
+
+## Render Deploy 🚀
+
+[Live demo here](https:)
+
```
Comment (KayLemba) [inline README.md]: Good job working on the project requirements but you forgot a very important requirements it says in the project requirements that:

Video presentation

Provide feedback about the video presentation in the code review summary.

The student recorded a video presenting the features of the project.
The maximum length of the video is 5 minutes.
The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
The video contains a demo of the project, highlighting all the project features.
The video contains a highlight about a piece of the code the student finds interesting or is proud of making.


Note: It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.

Live version


The student deploys the application using Render, Netlify or GitHub Pages.
The project is deployed properly, the home and detail page works as expected.
A link to the deployed project is present in the README.


It would be nice if you implemented the above requirements.

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @tanveerisonline   👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
Git flow is followed. ✔️
Nice UI design 
Tests are passing 

Required Changes ♻️
Check the comments under the review in the inline comments.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1 +1,161 @@
...
+<!-- PROJECT DESCRIPTION -->
+
+# Budget App <a name="about-project"></a>
+
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
+
+## video demo 🎥
+
+[video demo here](https://www.loom.com/share/3cd7d67c0f9640f790fbe998a70a4130)
```
Comment (sumairq) [inline README.md]: - [ ] Great job on adding the video presentation 👍, however according to the requirements, while demonstrating the project you have to verbally explain the project in such a way that is clearly audible in the video presentation 🔉 . 

- The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.

- The video contains a demo of the project, highlighting all the project features explained by you in professional english.

- The video contains a highlight about a piece of the code you find interesting or is proud of making explained by you in professional english.

To summarize please record a new video with you explaining the project in clear english... Cheers 👍

**Review by sumairq — CHANGES_REQUESTED** [review]
### Hi @tanveerisonline , 



Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 3]
```diff
@@ -1 +1,161 @@
...
+<!-- PROJECT DESCRIPTION -->
+
+# Budget App <a name="about-project"></a>
+
+> This mobile web app allows users to create categories and add transactions to each category. The app will show the total amount spent on each category. The app is built with Ruby on Rails. The app is deployed on Render.
+
+## video demo 🎥
+
+[video demo here](https://www.loom.com/share/c3197ca55f534b518e79cdd4a7b9b850)
+
```
Comment (cynthiainga) [inline README.md]: - [ ] Good so far @tanveerisonline but as mentioned before the maximum length of the video should be 5 minutes and no longer than 5 minutes, also you should appear in the video as mentioned in the project requirements 🙃 😉
- [ ] Please kindly record a new video that meets all the requirements 🙏 

Screenshot 2023-04-29 at 20 28 28

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @tanveerisonline, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Git flow is followed ✔️ 
- [ ] Nice design ✔️ 

 ## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
Hi @tanveerisonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Good design ✔️ 
- [ ] Correct Gitflow ✔️ 
- [ ] Good presentation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
